compiler/optimizations/forallOptimizations.cpp

```diff
@@ -144,10 +144,14 @@ Expr *preFoldMaybeLocalArrElem(CallExpr *call) {
   SymExpr *maybeArrElemSymExpr = toSymExpr(call->get(1));
   INT_ASSERT(maybeArrElemSymExpr);
 
-  SymExpr *controlSymExp = toSymExpr(call->get(2));
-  INT_ASSERT(controlSymExp);
+  SymExpr *controlSymExpr = toSymExpr(call->get(2));
+  INT_ASSERT(controlSymExpr);
 
-  bool confirmed = (controlSymExp->symbol() == gTrue);
+  SymExpr *ffControlSymExpr = toSymExpr(call->get(3));
+  INT_ASSERT(ffControlSymExpr);
```

Comment:
Could you add a comment next to the prim_def for PRIM_MAYBE_LOCAL_ARR_ELEM that describes what the arguments to the primitive mean?

Reply:
I had a comment at the beginning of this function, but it was outdated, I updated it.